backend/src/main/java/corea/auth/resolver/AccessedMemberArgumentResolver.java

```diff
@@ -30,8 +32,9 @@ public boolean supportsParameter(MethodParameter parameter) {
     public AuthInfo resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer,
                                     NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
         HttpServletRequest request = (HttpServletRequest) webRequest.getNativeRequest();
-        Member member = memberRepository.findByEmail(requestHandler.extract(request))
-                .orElse(null);
-        return member == null ? AuthInfo.ANONYMOUS : AuthInfo.from(member);
+        Optional<Member> memberOpt = memberRepository.findByEmail(requestHandler.extract(request));
```

Comment:
```suggestion
  return memberRepository
  .findByEmail(requestHandler.extract(request))
  .map(AuthInfo::from)
  .orElse(AuthInfo.getAnonymous());
        
```
굳이 Optional 로 받고 할 필요는 없는거 같은데 어떤지

Reply:
디미터 법칙에 위반하는 것 같아 그런거긴 합니다~
그럼에도 다른 분들도 저렇게 바꾸는게 더 좋다 느껴진다면 바꿀게요~

다른 분들 의견 주세용~ 👍